tests/test_ev_power_flap_fix.py

```
@@ -0,0 +1,110 @@
+"""EV flap fix (2026-07-10, Guido / KEBA P30): a UDP power blip must not
+collapse the EV budget.
+
+Root cause (proven): the KEBA charging-power sensor blips to ~0 for one
+cycle while the car is really drawing 10 kW. ``ev_power`` feeds the home
+energy balance (``home = solar + grid_import − ev − …``). While the fast
```

Ruff (0.15.20)

[warning] Docstring contains ambiguous `−` (MINUS SIGN). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)